metadata/root.json

```diff
@@ -2,17 +2,29 @@
  "signatures": [
   {
    "keyid": "c8e09a68b5821b75462ae0df52151c81deb7f1838246dc1da8c34cc91ec12bda",
-   "sig": "30440220658cdfd99f8920fabe6f7b3a8a25711d5bd2865f956562c74b4a964440bca1dc022029d3794d5db60e786197b4101266b06bc6d89eba839a406af95666494f4a2be7"
+   "sig": ""
```

Comment:
This is the blemish in the process: 
* we use empty sigs for signers to make git conflicts less likely (multiple signers can add sigs at same time because they all have their own lines to modify)
* if one of the signers does not sign, this results in an empty sig
* I consider this fine from spec perspective, but I also know some clients have had issues with this (sigstore-java for example still had an issue with this recently)
* We could actually automatically remove this empty sig before publishing -- I just don't like online-sign modifying root.json even if the change is not in the signed portion...  

Anyway: I think this is acceptable: if we find out clients have issues with it, we can avoid it in production